src/HavenoClient.test.ts

```diff
@@ -2600,6 +2600,11 @@ async function executeTrade(ctxP: Partial<TradeContext>): Promise<string> {
       ctx.taker.balancesAfterPayout = await ctx.taker.havenod?.getBalances();
     }
 
+    const trades = (await user1.getTrades()).concat(await user2.getTrades());
```

Comment:
Let's remove this here, and [here](https://github.com/haveno-dex/haveno-ts/blob/7192b899f493702547ce4ddce4dd1b5b3497f822/src/HavenoClient.test.ts#L2854) in takeOffer:

- Get user1 trades, assert the trade is contained by id, and call testTrade() on each.
- Get user2 trades, assert the trade is contained by id, and call testTrade() on each.